pkg/filesystem/subvolume.go

```diff
@@ -80,13 +80,13 @@ func getExternalClusterDetails(ctx context.Context, clientsets *k8sutil.Clientse
 
 	scList, err := clientsets.Kube.CoreV1().Secrets(clusterNamespace).List(ctx, v1.ListOptions{})
 	if err != nil {
-		logging.Fatal(fmt.Errorf("Error fetching secrets: %v\n", err))
+		logging.Fatal(fmt.Errorf("Error fetching secrets in namespace %q: %v", clusterNamespace, err))
 	}
 	for i := range scList.Items {
 		if strings.HasPrefix(scList.Items[i].ObjectMeta.Name, "rook-csi-cephfs-provisioner") {
 			data := scList.Items[i].Data
 			if data == nil {
-				logging.Fatal(fmt.Errorf("Secret data is empty"))
+				logging.Fatal(fmt.Errorf("Secret data is empty for %v/rook-csi-cephfs-provisioner", clusterNamespace))
```

Comment:
`%v` is used when printing a struct or other complex type. If it's just a string, please use `%s` when no quotes are needed. (same for other changes in this PR)
```suggestion
				logging.Fatal(fmt.Errorf("Secret data is empty for %s/rook-csi-cephfs-provisioner", clusterNamespace))
```